Application/EdFi.Ods.SandboxAdmin.Web/Global.asax.cs

```diff
@@ -100,5 +129,10 @@ private static void InitializeWebSecurity()
                 UsersContext.UserNameColumn,
                 autoCreateTables: true);
         }
+
+        public class PostgresSimpleRoleProvider : SimpleRoleProvider
+        {
+
+        }
```

Comment:
do we need this?

Reply:
ahh i guess i added it thinking i was going ot need to something for postgres, i will remove.